src/saveload.cpp

```diff
@@ -76,6 +76,12 @@ void save_texture(const void *data, uint32_t dataSize, uint32_t width, uint32_t
 	char filename[sizeof(basedir) + 1024];
 	uint64_t hash;
 
+	if (!save_textures) {
+		ffnx_warning("Save texture skipped because the option \"save_textures\" is disabled (name=%s).\n", name);
```

Comment:
This will create another massive log statement, so I'd suggest prefixing it with another if:
```suggestion
if (trace_all) ffnx_warning("Save texture skipped because the option \"save_textures\" is disabled (name=%s).\n", name);
```
Fine to add other `trace_` vars if needed for this case.

Reply:
At first no, this log will not appear as everytime we use save_texture, we check if save_textures is enabled before. In the future maybe, if we forgot to check before

Reply:
Ah ok I see now the angle of this. I thought you wanted to short circuit it in the function itself, but if it's supposed to be just a warning while we develop makes sense :) Thanks!